root/etc/systemd/system/cockpit.service.d/nethserver.conf

```
@@ -0,0 +1,2 @@
+[Service]
+EnvironmentFile=-/etc/sysconfig/cockpit
```

This shouldn't be effective until next `systemctl daemon-reload`. However I wouldn't fix it because it is required just once, and it surely occurs when one of the following things happens:

- system reboot
- an RPM containing a systemd unit is updated

As the current Cockpit policy is far from being considered unsecure I'd not worry...

```suggestion
EnvironmentFile=-/etc/sysconfig/cockpit
```

---

just trying to reproduce with a fresh install @DavidePrincipi 

install the rpm of this PR
test the cipher, tls 1.1 and 1.2 are allowed
move to the policy 20200510
wait (a long time) the service restart alone, about 5 minutes. You can try to login root several time
test the cipher suite, we are only allowed to tls1.2

It seems we have nothing more to do indeed